book/src/sprites/display_the_texture.md

```diff
@@ -3,31 +3,31 @@
 With a [`DrawFlat2D`][doc_drawflat2d] render pass set up and a loaded texture, it's already possible to render the full texture. The [`TextureHandle`][doc_tex_handle] itself implements [`Component`][doc_component], so you can just attach the [`Handle`][doc_handle] to an entity and it'll show up!
 
 For anything rendered by the [`DrawFlat2D`][doc_drawflat2d] pass, it's also possible to optionally attach a [`Flipped`][doc_flipped] component to the entity, which will signal to the renderer that you want to flip your texture horizontally or vertically when rendering.
+TODO
```

Comment:
I didn't how this works now.

Reply:
Flipping sprites is now done in one of two methods:

* By passing `true` to `Sprite::from_pixel_values` for the `flip_horizontal` or `flip_vertical` parameters when building the sprite sheet.
* At runtime, by changing the y or x rotation on the entity's `Transform`.

    ```rust
    use amethyst::core::math::RealField;

    transform.set_rotation_y_axis(f32::pi()); // Flip horizontally
    transform.set_rotation_x_axis(f32::pi()); // Flip vertically
    ```

Reply:
It doesn't really suit the current flow of the text to mention sprite sheet loading though.